riva/client/tts.py

```diff
@@ -34,6 +34,10 @@ def synthesize(
         language_code: str = 'en-US',
         encoding: AudioEncoding = AudioEncoding.LINEAR_PCM,
         sample_rate_hz: int = 44100,
+        audio_prompt_file: Optional[str] = None,
+        audio_prompt_encoding: AudioEncoding = AudioEncoding.LINEAR_PCM,
+        audio_prompt_sample_rate: int = 22050,
```

Comment:
can we infer sample rate from the file and not require user to provide `audio_prompt_sample_rate` ?

Reply:
Done

Reply:
@virajkarandikar good to resolve?